bower.json

```diff
@@ -1,38 +1,37 @@
 {
   "name": "paper-pagination",
-  "version": "0.2.2.1",
-  "description": "A material design element that generate pagination bar",
+  "version": "2.0.2",
+  "description": "A material design element inspire by Danny Phung that generate pagination bar",
   "authors": [
-    "Dong Hung Phung"
+    "Benoit Claveau <benoit.claveau@gmail>"
   ],
   "keywords": [
-    "web-components",
-    "web-component",
+    "webcomponents",
     "polymer",
     "paper",
     "pagination",
-    "paper-pagination",
-    "pagin"
+    "paper-pagination"
   ],
   "main": "src/paper-pagination.html",
   "repository": {
     "type": "git",
     "url": "git://github.com/PolymerElements/paper-material"
   },
-  "license": "https://opensource.org/licenses/MIT",
-  "homepage": "https://github.com/dannyphung/paper-pagination/",
+  "license": "MIT",
+  "homepage": "https://dannyphung.github.io/paper-pagination",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.1.0",
-    "paper-button": "polymerelements/paper-button#^1.0.0",
-    "paper-menu": "polymerelements/paper-menu#^1.0.0",
-    "paper-item": "polymerelements/paper-item#^1.0.0"
+    "polymer": "Polymer/polymer#^2.5.0",
+    "paper-button": "polymerelements/paper-button#^2.0.0",
+    "paper-listbox": "polymerelements/paper-listbox#^2.0.0",
+    "paper-item": "polymerelements/paper-item#^2.0.0",
+    "web-animations-js": "^2.3.1"
```

Comment:
since you've added `neon-animation` in paper-pagination.html:5, you should add it to the bower file as well

Reply:
Hi dannyphung, I replace neon-animation by web-animaton because I need to control thé animation in js.

Reply:
if like that then you should remove the import line also.